db/fixtures/miq_product_features.yml

```diff
@@ -3270,9 +3270,19 @@
           :hidden: true
           :identifier: db_refresh
 
+# Documentation
+- :name: Documentation
+  :description: Open the documentation page
+  :feature_type: node
```

Comment:
@skateman I just looked at other items in the file and the Descriptions are normally done like titles, so these should be:
* Open the Documentation Page
* Open the Product Website
* Show the About Information